sdks/java/core/src/test/java/org/apache/beam/sdk/io/LocalFileSystemTest.java

```diff
@@ -184,6 +184,27 @@ public void testDelete() throws Exception {
     assertTrue(f3.exists());
   }
 
+  @Test
+  public void testMatchWithGlob() throws Exception {
+    String globPattern = "/A/a=[0-9][0-9][0-9]/*/*";
+    File folder1 = temporaryFolder.newFolder("A/a=100/data1");
+    File folder2 = temporaryFolder.newFolder("A/a=233/data_dir");
+    File expectedFile1 = new File(folder1, "/file1");
```

Comment:
you don't need the "/" in the file names.

Reply:
Of course, not sure why its there, kind of embarrassing :)